js/packages/openinference-vercel/src/AISemanticConventions.ts

```diff
@@ -13,7 +13,7 @@ const AIPrefixes = {
   telemetry: "telemetry",
   prompt: "prompt",
   toolCall: "toolCall",
-  result: "result",
+  result: "response",
```

Comment:
```suggestion
  response: "response",
```
Shouldn't this be the case here? @Parker-Stafford

Reply:
yeah for consistency lets do that, then you'll want to change these
https://github.com/Arize-ai/openinference/blob/main/js/packages/openinference-vercel/src/AISemanticConventions.ts#L53-L60
 to response as well and key off `AIPrefixes.response`

Reply:
and these
https://github.com/Arize-ai/openinference/blob/main/js/packages/openinference-vercel/src/utils.ts#L460-L471

Reply:
Thanks for the suggestions @Parker-Stafford and @mikeldking! I went through and changed result to response throughout the package